Source/WebCore/rendering/RenderDeprecatedFlexibleBox.cpp

```diff
@@ -202,6 +202,8 @@ void RenderDeprecatedFlexibleBox::computeIntrinsicLogicalWidths(LayoutUnit& minL
     };
 
     if (shouldApplySizeContainment()) {
+        if (auto width = explicitIntrinsicInnerLogicalWidth())
+            minLogicalWidth = maxLogicalWidth = width.value();
```

Comment:
These two lines happen a lot, could we have a shared method?

Reply:
Ok, I misread, I thought we were setting member vars here. I don't think we can improve much here.